frontend/src/components/navigation/Navbar.jsx

```diff
@@ -1,14 +1,15 @@
 // NPM Packages
 import React from "react";
 import { Link } from "react-router-dom";
+import SearchBar from "../search/SearchBar";
```

Comment:
Pls could you add an empty line after line 3 and then add `// Project files` above the SearBar import?